org.tildearrow.Furnace.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.tildearrow.Furnace
```

Comment:
the appdata say `  <id>org.tildearrow.furnace</id>`

You should is it instead.